HeterogeneousCore/AlpakaTest/plugins/BuildFile.xml

```diff
@@ -0,0 +1,29 @@
+<!-- host-only plugins -->
+<library file="*.cc" name="HeterogeneousCoreAlpakaTestPlugins">
+  <use name="alpaka"/>
+  <use name="DataFormats/PortableTestObjects"/>
+  <use name="FWCore/Framework"/>
+  <use name="FWCore/MessageLogger"/>
+  <use name="FWCore/ParameterSet"/>
+  <use name="FWCore/Utilities"/>
+  <use name="HeterogeneousCore/AlpakaCore"/>
+  <use name="HeterogeneousCore/AlpakaInterface"/>
```

Comment:
Is `AlpakaCore` really needed for the host-only plugins? It seems to me the explicit dependence on `AlpakaInterface` could be removed as well.

Reply:
For an alpaka-based host-only plugin, I think it would still be needed.

But in fact, `TestAlpakaAnalyzer` is not based on alpaka at all, so we can remove all dependencies here.